adapters/handlers/rest/handlers_objects_test.go

```diff
@@ -984,14 +984,58 @@ func TestEnrichObjectsWithLinks(t *testing.T) {
 			t.Errorf("expected: %T got: %T", objects.ObjectsClassReferencesDeleteUnprocessableEntity{}, res)
 		}
 	})
+
+	t.Run("Query", func(t *testing.T) {
+		var (
+			cls = "MyClass"
+			m   = &fakeManager{
+				queryErr: nil,
+				queryResult: []*models.Object{{
+					Properties: map[string]interface{}{"name": "John"},
+				}},
+			}
+			h   = &objectHandlers{manager: m, logger: &logrus.Logger{}}
```

Comment:
Just a suggestion for future.
In tests we tend to use `test.NullLoger()` from `"github.com/sirupsen/logrus/hooks/test"` package instead of `&logrus.Logger{}`

Reply:
Thanks for the hint. I would use it next time